test/pjrt/test_experimental_pjrt_multi_cpu.py

```diff
@@ -108,6 +108,23 @@ def test_spawn(self, spawn):
     indices = sorted(queue.get(block=False) for _ in range(queue.qsize()))
     self.assertListEqual(indices, list(range(4)))
 
+  @staticmethod
+  def _hlo_dump(tmpdir: str):
+    os.environ['XLA_SAVE_TENSORS_FMT'] = 'hlo'
+    os.environ['XLA_SAVE_TENSORS_FILE'] = os.path.join(tmpdir, 'save.hlo')
```

Comment:
hmm, I actually don't know if `os.environ` is local to this function. Will `os.environ` got reset upon return of this function?

Reply:
It's not local, but we only change `os.environ` in the sub process, so it does get reset between tests